SmartDeviceLink/private/SDLVoiceCommandManager.m

```diff
@@ -109,15 +110,21 @@ - (void)setVoiceCommands:(NSArray<SDLVoiceCommand *> *)voiceCommands {
         return;
     }
 
+    // Set the new voice commands internally
+    _voiceCommands = [self sdl_removeEmptyVoiceCommands:voiceCommands];
```

Comment:
I'm seeing a couple of failing test cases. Due to the fact the `voiceCommands` instance var is set with a new object returned by `sdl_removeEmptyVoiceCommands`, the check to compare `if (voiceCommands == self.voiceCommands) { ... }` always fails.